_posts/blog/2018-09-18/2018-09-18-5 -tips-to-make-your-openbazaar-listings-pop.markdown

```diff
@@ -0,0 +1,158 @@
+
+For example, instead of “red”, try:
+
+- _Vermillion_
+- _Crimson_
+- _Raspberry_
+- _Burgundy_
+
+> **TIP:** If you’re stuck on words, check out [Ingrid Sundberg’s Colour Thesaurus](http://ingridsundberg.com/2014/02/04/the-color-thesaurus/)
+
+![Color Chart for OpenBazaar](Color Chart for OpenBazaar.png "Color Chart for OpenBazaar")
+*Image from Ingrid Sundberg’s Colour Thesaurus*
+
+#### Size
+
+Bust out the tape measure so you can let people know how big or small your item is. If you’re selling clothing, it’s helpful to include a size guide like this:
+
+![Example of a Size Guide on OpenBazaar](Example of a Size Guide on OpenBazaar.png "Example of a Size Guide on OpenBazaar")
+
+#### Material
+  
+What’s your product made of? Stainless steel, copper, wood, bamboo… If this lip balm features beeswax collected from your own personal backyard apiary, be sure to let your buyers know!
+
+#### How to use 
+
+Give us some instructions on how to use and care for your product. For example: Where should it be stored? Is it dishwasher safe or hand wash only? Can I toss it in the dryer?
+
+Inspire your buyers. Show them different ways to use your product. If you’re selling a scarf, offer a few examples for how they can wear and style that scarf. 
+
+On top of that, you can also give them some ideas for how a product might fit into their lives --
+
+- _This kindle would be a great gift for your neighborhood bookworm who may or may not have a hoarding problem!_
+- _Does your cat take massive poops? Our double-duty clothespin doubles as a nostril clamp!_
+
+Absolutely irresistible.
+
+### 4. Break up long blocks of text
+
+Product descriptions or store policies can get lengthy. Don’t make your buyer slog through line after line of text. Instead, format your content to help them find the information they need.
+
+#### Write short paragraphs
+
+Keep your paragraphs to 3-5 lines max. If you have a lot to communicate, try organizing the content into sections with header titles. It will make the information much less intimidating and easier to digest. For example, in your product description, you can have sections for _key features_, _dimensions_, or _care instructions_.
+
+#### Use bullet points
+
+Bullet points are your friends! They’re easy to read and convey a lot of information very quickly. Use them when you can. They’re especially helpful when listing product features.
+
+#### Format your text
+
+A little text formatting can go a long way. Call out important tidbits in **bold** or _italics_ so they don’t get lost.
+
+#### Add visuals 
+
+Get creative! Break up your text with photos and GIFs. Visuals can add an instant POP to your content, making it dynamic and engaging. For example, check out [this listing by Crypto Republic - BCH](https://openbazaar.com/store/QmU5ZSKVz2GhsqE6EmBGVCtrui4YhUXny6rbvsSf5h2xvH/mini-metro-cd-key-steam).
```

Comment:
Should we mention that you'll need a photo host here? OB doesn't store images that are in the description, you have to store them somewhere on the normal internet and link the URL in your text to display them.

Reply:
Yes. Added:

**Note:** OpenBazaar doesn't store images that you can include in the description. You need to store them somewhere else on the internet and use their URL to to add them.